dcbr-web/src/components/RenewalForm.vue

```diff
@@ -87,7 +87,7 @@ export default {
     ],
     registrationNumberRules: [
       v => !!v || "Registration number is required",
-      v => v.length <= 50 || "Registration number must be less than 50 characters"
+      v => v.length <= 20 || "Registration number must be less than 20 characters"
```

Comment:
Can we do a pattern/mask matching here to ensure the registration number they enter is in the format we expect? Like `DCBR-123456` or whatever it is that we generate?